lib/loader/index.js

```diff
@@ -47,6 +57,27 @@ function validateArguments (url, options, callback) {
 
 module.exports = {
   /**
+   * Clears the loaded URLs
+   *
+   * Clears any URLs that might have been stored in the loadedUrls hash, either
+   * locally or in the given namespace(optional). This is needed for testing so
```

Comment:
If you absolutely must, then I suggest marking it as private `_clearLoadedUrls` or `_cleanup` or something that indicates it's not part of the public API of this module.